src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -3004,6 +3004,16 @@ private StatementAst ConfigurationStatementRule(IEnumerable<AttributeAst> custom
                         dropIntoDebugger: true);
                 }
 
+                // Configuration is not supported on ARM64
+                if (PsUtils.IsRunningOnProcessorArchitectureARM())
+                {
+                    ReportError(configurationToken.Extent,
```

Comment:
Can you fix the formatting here to comply with CodeFactor?

Reply:
Fixed.